app/src/main/java/com/example/sports/ui/theme/Theme.kt

```diff
@@ -14,63 +14,150 @@
  * limitations under the License.
  */
 
-package com.example.sports.ui.theme
+package com.example.sports.ui
```

Comment:
Fix the package name `com.example.sports.ui.theme`
So you don't have to import all the colors, they are from the same package.